recipes/kwant/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "kwant" %}
+
+package:
+  name: {{ name }}
+  version: "1.2.2" [py3k]
```

Comment:
The linter is choking here. You can lint a recipe locally with `conda smithy recipe-lint`.